engineering/README.md

```diff
@@ -0,0 +1,155 @@
+- [ ] Is there properly formatted (eg. jsdoc for JavaScript) documentation for public and private API methods?
+- [ ] Does any code duplicate functionality?
+- [ ] Are there any global variables that can be removed?
+- [ ] Is there any commented out code?
+- [ ] Is there any logging or debugging code that isn't using a logger that can be removed?   
+- [ ] Do tests exist and do they adequately cover the changed behavior?
+- [ ] Is the code being tested by any linter or type checking tools and does it pass?
+
+
+## Repository management
+We generally use the [git flow](http://nvie.com/posts/a-successful-git-branching-model/) git repository workflow. 
+
+The `develop` branch contains the latest delivered and accepted features. New features are generally developed against the `develop` branch unless they depend on the work done in another feature branch that has yet to be accepted and merged into `develop.
+
+## Planning
+[ ] Before Friday Engineering meeting: Each team member collects notes on their top priorities for the coming week, progress on existing items, concerns etc. and adds bullet list items to Friday engineering meeting agenda.
+[ ] Friday Engineering meeting: team members discuss their priorities and concerns along with those of the community.
+[ ] Meeting deliverable: consensus on a few bullet point priorities for the following week that will be presented to co-founders. These should be listed at the top of the agenda document and link to github issues if they exist.
+[ ] Co-founders & team members consider these proposed engineering priorities before the weekly team meeting.
+[ ] Co-founders present their priorities for the upcoming week having taken into account the engineering teams priorities & team discusses.
+
+## Meetings
+
+Meetings are expensive, especially for creative people like designers and engineers. A 15 minute meeting can mean an hour or more of lost productivity. Meetings are also synchronous and mean that only those that can be available at that certain time have a chance to give input into the discussion. As such, we prefer to default to conducting discussions on github issues or in the forum and to minimize meetings whenever possible.
+
+### Meeting rules
+
+[ ] Meetings always begin on time: starting late penalizes the punctual and rewards the tardy. You should connect to the meeting conference call or arrive in the conference room a few minutes before the start time.
+[ ] Meetings always end on time: our team member and community member time is very valuable and they have other commitments. Disciplined meetings can cover the required topics within the allocated time.
+[ ] No meeting shall proceed without a written agenda: if it's not important enough to create an agenda, it's not important enough to hold a meeting
+[ ] Each meeting should have a stated goal, deliverable or outcome. This helps keep the meeting on track.
```

Comment:
To add to the rules, should we also say that "Each meeting has a designated note-taker, who will publish the notes on the forum shortly after the meeting ends?"

Reply:
yep - added a note about that